### DISCUSSION TITLE
How to plot Cumulative kappa using frequency as x axis rather than mfp?

### DISCUSSION THREAD

**Discussion by abchaoming1** [ttadano/alamode#184] [Q&A]
Dear developers and users,
     I am using ALAMODE to calculate the cumulative kappa with analyze_phonons.py code. I found no options or prompts to use frequency as the x-axis for plotting the cumulative kappa in the analyze_phonons.py(maybe I missed something? It is all about the mfp as the x-axis). How can I employ the frequency as the x-axis to plot?
     
Thanks!

**Comment by Lamed-git** [comment]
I think the Thermal conductivity spectrum may help you solve this problem. If you integrate the Thermal conductivity spectrum, the resulting outcome should be Cumulative kappa vs frequency.
